src/OrchardCore/OrchardCore.XmlRpc.Abstractions/XmlRpcContext.cs

```diff
@@ -12,6 +12,6 @@ public class XmlRpcContext
         public IUrlHelper Url { get; set; }
         public XRpcMethodCall RpcMethodCall { get; set; }
         public XRpcMethodResponse RpcMethodResponse { get; set; }
-        public ICollection<IXmlRpcDriver> Drivers = [];
+        public ICollection<IXmlRpcDriver> Drivers { get; set; } = [];
```

Comment:
Why do you need this change too?

```suggestion
        public ICollection<IXmlRpcDriver> Drivers { get; init; } = [];
```